src/mono/wasm/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -834,6 +836,33 @@ protected void AssertBlazorBundle(string config, bool isPublish, bool dotnetWasm
                         dotnetJsPath!,
                         "Expected dotnet.native.js to be same as the runtime pack",
                         same: dotnetWasmFromRuntimePack);
+
+            string bootConfigPath = Path.Combine(binFrameworkDir, "blazor.boot.json");
+            Assert.True(File.Exists(bootConfigPath), $"Missing '{bootConfigPath}'");
+
+            using (var bootConfigContent = File.OpenRead(bootConfigPath))
+            {
+                var bootConfig = ParseBootData(bootConfigContent);
+                foreach (var dotnetJs in bootConfig.resources.runtime.Keys.Where(k => k.StartsWith("dotnet.") && k.EndsWith(".js")))
+                {
+                    Assert.DoesNotContain(dotnetJs, "..");
```

Comment:
It would be preferable to extract the string between `dotnet.`, and `.js`, and check that for being non-zero length. If not, add a comment here explaining the reasoning for checking `".."`.

Reply:
What are the possible version patterns? 
I can think of `8.0.0`, `8.0.0-dev` and `8.0.0-preview.5.123.1`. Do you know of any other?

Reply:
Thanks, I wasn't sure if there is any other "pattern" for version number